### ISSUE TITLE
Glow.lua error

### ISSUE THREAD

**Issue by TreizeEU** [WeakAuras/WeakAuras2#1815]
**Describe the bug**

With the latest alpha release, I get the below lua error and the addon is completely broken. To be able to change settings or add/remove strings (and get strings to work), I have to move the glow.lua file to my desktop. By doing so causes the addon to work but spit out error of not finding the mentioned file.

**Do you have an error log of what happened?**

44x WeakAuras\SubRegionTypes\Glow.lua:164: attempt to index field 'button' (a nil value)
WeakAuras\SubRegionTypes\Glow.lua:164: in function `SetVisible'
WeakAuras\SubRegionTypes\Glow.lua:304: in function `modify'
...aceWeakAuras\RegionTypes\RegionPrototype.lua:512: in function `modifyFinish'
WeakAuras\RegionTypes\Icon.lua:543: in function `modify'
WeakAuras\WeakAuras-2.15.5-16-g176ab43.lua:4333: in function `SetRegion'
WeakAuras\WeakAuras.lua:4240: in function <WeakAuras\WeakAuras.lua:4155>
WeakAuras\WeakAuras.lua:4264: in function `Add'
WeakAuras\WeakAuras.lua:3815: in function `load'
WeakAuras\WeakAuras.lua:3822: in function `AddMany'
WeakAuras\WeakAuras.lua:1534: in function <WeakAuras\WeakAuras.lua:1520>

WeakAuras\WeakAuras-2.15.5-16-g176ab43.lua:1661: in function <WeakAuras\WeakAuras.lua:1613>

**To Reproduce**

1. Update to latest version (WeakAuras-2.15.5-16-g176ab43)
2. Log into the game.
3. See error

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**

No, but I did test with empty weakaura (as in, I moved all 4 WA.wtf files to my desktop and launched the game) and got no lua errors.

I also updated all my strings (and removed some legacy and homemade strings) to their respective version (none of which have been updated since this latest version of WA was put up) but that didn't solve the issue.

**Which version of WeakAuras are you using?**

2.15.5-16-g176ab43

**Was it working in a previous version? If yes, which was the last good one?**

2.15.5-6-g9e2e1ca

**Additional context**

Attaching all 4 \SavedVariables\WeakAuras.lua +.bak and \SavedVariables\WeakAurasOptions.lua +.bak in that zip file because I don't know if they're helpful.

[WeakAuras.zip](https://github.com/WeakAuras/WeakAuras2/files/3846869/WeakAuras.zip)

**Comment by InfusOnWoW** [comment]
Thanks, that log looks odd, I'll try to reproduce it later today.

**Comment by InfusOnWoW** [comment]
I can't reproduce the issue, can you try whether it reproduces if you disable all addons? And as a second test, if you disable all other addons except WeakAuras and Masque.

The code path where the error is happening is inside the masque support code, so the first test should not show the error and the second test might show the error again.

**Comment by TreizeEU** [comment]
I first tested by turning off Masque, which resulted in no errors and a working weakaura, so I assume that's what caused it. I am using an older version of Masque (5.0.387) from 2014 which is the last version that still lets me use the old ButtonFacade_Rothar skin (as far as I know, I haven't updated Masque since then).

I also decided to back-up those ButtonFacade folders and then update Masque to latest version, then copy back the folders again, no errors now and I can keep using the old Rothar skin. :D
